### ISSUE TITLE
Route metrics

### ISSUE THREAD

**Issue by grzegorz-gutowski** [OpenVPN/openvpn3-linux#234]
Hello,
It seems that openvpn3 silently discards metric specification on route commands:
```--route network/IP [netmask] [gateway] [metric]```
and creates all routes with default metric.
I'm using `metric` in my pushed routes and openvpn2 linux clients handle metric specification correctly.
Should I stop relying on this behavior?

**Comment by dsommers** [comment]
This is clearly a missing feature.  I'm working my way through this code, as part of #171.  The whole "network configuration service" (`openvpn3-service-netcfg`) is being ported to a new D-Bus implementation.  Once that is done, we can look into how to improve route metric handling.

The current v22_dev work in progress is already quite large and intrusive, so I'll postpone this for a following release.
